plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -82,6 +85,19 @@ public PinotConfig setControllerUrls(String controllerUrl)
         return this;
     }
 
+    public Optional<URI> getBrokerUrl()
+    {
+        return brokerUrl;
+    }
+
+    @Config("pinot.broker-url")
+    @ConfigDescription("Provide global Broker URL. Setting this property will disable broker discovery mechanism.")
+    public PinotConfig setBrokerUrl(URI brokerUrl)
```

Comment:
We only use host and port in the provided value. It doesn't make sense to me.

Reply:
`HostAndPort` (from Guava) is a good alternative - we seem to already use it in a few places.

Reply:
yes confirmed we can use HostAndPort

Reply:
Change to `HostAndPort`. Wondering if property name should still have `-url` in it, decide to keep it.